src/shared/Core/Interop/Windows/WindowsFileSystem.cs

```diff
@@ -7,8 +7,16 @@ public class WindowsFileSystem : FileSystem
     {
         public override bool IsSamePath(string a, string b)
         {
-            a = Path.GetFileName(a);
-            b = Path.GetFileName(b);
+            if (string.IsNullOrWhiteSpace(a) || string.IsNullOrWhiteSpace(b))
+            {
+                return false;
+            }
+
+            a = Path.GetFullPath(a);
+            b = Path.GetFullPath(b);
+
+            // Note: we do not resolve or handle symlinks on Windows
+            // because they require administrator permissions to even create!
```

Comment:
That's not true on Windows 10 in Developer Mode, though.

Reply:
But until there's enough of the world on a version of Windows where you don't need admin to create a symlink, then it's not worth our time supporting symlinks at all in Windows, IMO.

Reply:
We do not have _any_ reliable indicator in Git for Windows as to how many users run in Developer Mode and use symlinks, but there have been enough complaints that we had to implement patches to support that (git-for-windows/git@9086edc588a7e88d9b071fbf030436e5919dd58b, git-for-windows/build-extra@e6085cb6248c10584d0c43168edecf4cf914b556, just to name two).

So I do not believe that the fraction of symlink/Developer Mode users is actually small.

Reply:
But how many people are going to be installing and configuring standalone GCM via a symlink? :-)

Reply:
Thanks for the context however on symlink usage for G4W 👍